src/panel/direction/MobileRouteDetails.jsx

```diff
@@ -18,22 +18,20 @@ const MobileRouteDetails =
   >
     <div className="mobile-route-details-header">
       <div className="mobile-route-details-header-content">
-        <Flex alignItems="center" justifyContent="space-between" className="u-text--title">
-          {formatDuration(route.duration)}
-          <CloseButton onClick={() => { toggleDetails(id); }} />
+        <Flex alignItems="flex-start" justifyContent="space-between">
+          <RouteSummaryInfo
+            route={route}
+            vehicle={vehicle}
+          />
+          <CloseButton onClick={() => toggleDetails(id)} />
```

Comment:
What about using the prop `position="topRight"`, sot it looks better positionned with the content paddings.